protocol/rest/rest_server/gin_server.go

```diff
@@ -0,0 +1,46 @@
+package rest_server
+
+import (
+	"context"
+	"github.com/apache/dubbo-go/common"
+	"github.com/apache/dubbo-go/common/logger"
+	"github.com/gin-gonic/gin"
+	"net/http"
+	"time"
+)
+
+type GinServer struct {
+	engine *gin.Engine
+	srv    *http.Server
+}
+
+func (gs *GinServer) Start(url common.URL) {
+	srv := &http.Server{
+		Addr:    ":8080",
+		Handler: gs.engine,
+	}
+	srv.ListenAndServe()
+}
+
+func (gs *GinServer) Deploy(service interface{}, config interface{}) {
+	//TODO gin http 部署接口
+	// handler中处理：
+	// http server收到http request，转交给server；
+	// server根据RestService metadata找到Service，并将请求反序列化，构造好Service调用的上下文；
+	// RequestMapper这时候将不会处理什么事情，而是直接转交给Service的实现。
```

Comment:
don't use chinese comment

Reply:
I will remove it, when I finish rest protocol export function.